### ISSUE TITLE
manifests.yml lists 0.4.0 release but that release does not exist on github

### ISSUE THREAD

**Issue by mhansonp** [Netflix/ndbench#87]
While attempting to push and run this app in a CF environment, we got an error stating that 
ndbench-web-0.4.0-SNAPSHOT.war didn't exist and once checked we noted that ndbench-web-0.3.11-SNAPSHOT.war had been downloaded. It appears that the 0.4.0 release is not available even though it is referred to in the manifests.yml.

**Comment by ipapapa** [comment]
I do not believe we have a 0.4.0 release: https://github.com/Netflix/ndbench/releases The latest one I created is 0.3.11.
